iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/mapper/HouseTableMapper.java

```diff
@@ -20,7 +20,6 @@ public abstract class HouseTableMapper {
   @Autowired FileIOManager fileIOManager;
 
   @Mapping(target = "lastModifiedTime", ignore = true)
-  @Mapping(target = "creationTime", ignore = true)
```

Comment:
https://github.com/linkedin-multiproduct/openhouse/pull/476
You didn't want them to come into userTable. But since userTable does not have these fields, the lines are redundant.